tasks/bumppom.js

```diff
@@ -90,6 +90,15 @@ module.exports = function(grunt) {
 
     var _total = src.length;
     var _backup_dest = '';
+    
+    var fnReplaceSnapshot = function(version){
+      return (options.use_snapshot) ? version.replace(/\-.*$/, '-SNAPSHOT') : version;
```

Comment:
I might me missing something here, but would this not replace every character after the last `-` ? Don't you want to differentiate the snapshots?